packages/commonwealth/client/scripts/app.ts

```diff
@@ -655,7 +655,7 @@ Promise.all([$.ready, $.get('/api/domain')]).then(
             }),
             // Notifications
             '/notification-settings': importRoute(
-              'views/pages/notification_settings',
+              'views/pages/notification_settings/index',
```

Comment:
i don't believe this is necessary? stylistic sugar question but pushback is fine.